data/operators/power/substation.json

```diff
@@ -8363,7 +8350,8 @@
       "matchNames": [
         "enso",
         "enso netz",
-        "sachsennetze"
+        "sachsennetze",
```

Comment:
Please check whether the matchName is correct, see https://www.wikidata.org/wiki/Q124370321